### DISCUSSION TITLE
Adding SBS chanting book

### DISCUSSION THREAD

**Discussion by Devamitta** [bksubhuti/tipitaka-pali-reader#250] [Q&A]
Bhante,

Is it possible to add the "SBS Pali-English recitation book" to the TPR extensions?

Original repo of SBS book:

https://github.com/sasanarakkha/pali-english-recitations

Latest update of SBS book, exported in various formats:

https://github.com/sasanarakkha/pali-english-recitations/releases/latest

**Comment by bksubhuti** [comment]
this has been moved to 
https://github.com/bksubhuti/tpr_downloads/issues/5
